src/App.tsx

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component {
```

Comment:
maybe it would be better to delete 'React.', leave just 'Component' and add types?

Reply:
Yep, add generic typing